pyaedt/emit.py

```diff
@@ -45,44 +271,50 @@ class Emit(FieldAnalysisEmit, object):
         Whether to release AEDT on exit. The default is ``False``.
     student_version : bool, optional
         Whether to open the AEDT student version. The default is ``False``.
+    port : int, optional
+        Port number on which to start the oDesktop communication on an already existing server.
+        This parameter is ignored when creating a new server. It works only in 2022 R2 or later.
+        The remote server must be up and running with the command `"ansysedt.exe -grpcsrv portnum"`.
     machine : str, optional
         Machine name to which the oDesktop session is to connect to. This
         parameter works only in 2022 R2 and later. The remote server must be
         up and running with the command `"ansysedt.exe -grpcsrv portnum"`.
         If the machine is `"localhost"`, the server starts if it is not present.
-    port : int, optional
-        Port number on which to start the oDesktop communication on an already existing server.
-        This parameter is ignored when creating a new server. It works only in 2022 R2 or
-        later. The remote server must be up and running with the command `"ansysedt.exe -grpcsrv portnum"`.
     aedt_process_id : int, optional
         Process ID for the instance of AEDT to point PyAEDT at. The default is
         ``None``. This parameter is only used when ``new_desktop_session = False``.
 
     Examples
     --------
-    Create an instance of Emit and connect to an existing Emit
-    design or create a new Emit design if one does not exist.
-
+    Create an instance of Emit. You can also choose to define parameters for Emit here.
     >>> from pyaedt import Emit
-    >>> app = Emit()
-
-    Create a instance of Emit and link to a project named
-    ``"projectname"``. If this project does not exist, create one with
-    this name.
-
-    >>> app = Emit(projectname)
+    >>> aedtapp = Emit()
 
-    Create an instance of Emit and link to a design named
-    ``"designname"`` in a project named ``"projectname"``.
+    Typically, it is desirable to specify a project name, design name, and other parameters.
+    >>> aedtapp = Emit(projectname, designame)
 
-    >>> app = Emit(projectname,designame)
+    Once an instance of Emit is initialized, the schematic can be edited:
+    >>> rad1 = aedtapp.modeler.components.create_component("Bluetooth")
+    >>> ant1 = aedtapp.modeler.components.create_component("Antenna")
+    >>> if rad1 and ant1:
+    >>>     ant1.move_and_connect_to(rad1)
 
-    Create an instance of Emit and open the specified project,
-    which is named ``Myfile.aedt``.
-
-    >>> app = Emit("myfile.aedt")
+    Once the schematic is generated, the ``Emit`` object can be analyzed to generate
+    a revision. Each revision is added as an element of the ``Emit`` object member's
+    ``revisions_list``.
```

Comment:
```suggestion
   revisions list.
```
Descriptions should try to avoid literal tags. The sentence should be complete and in regular English (no code) if possible.